### PR TITLE
telemetry: add fix rate metrics for hover events

### PR DESCRIPTION
**Pull request by ctlai95**
## Problem

We want to capture ACR fix generation rate.

## Solution

Report the ratio of hover events for issues with and without fix.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by justinmk3** [comment]
failing test is relevant. is this expected for today's release?

**Comment by justinmk3** [comment]
waiting for review from a cw team member

**Review by laileni-aws — APPROVED** [review]

**Review by mj632 — APPROVED** [review]

File: .changes/next-release/Feature-acb30000-5eeb-4a1a-85ce-1df104b9cc86.json
Comment (nkomonen-amazon) [inline .changes/next-release/Feature-acb30000-5eeb-4a1a-85ce-1df104b9cc86.json:3]: No need for a changelog item since this change is not user facing

Comment (ctlai95) [reply .changes/next-release/Feature-acb30000-5eeb-4a1a-85ce-1df104b9cc86.json:3]: sounds good, removing it

**Review by nkomonen-amazon — APPROVED** [review]

**Pull request merged**
